Support/Scripts/start-android-emulator

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import os
+import argparse
+import subprocess
+from typing import Dict, Any, Tuple
+import sys
+import copy
+
+
+def run_argument_parser() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--target", help="the target emulator to run")
+    parser.add_argument(
+        "--download-path", help="the directory where the ndk and sdk are installed"
+    )
+
+    args = parser.parse_args()
+    return args
+
+
+def clean_arguments(args: argparse.Namespace) -> argparse.Namespace:
```

Comment:
I think this is messy because of the stupid stuff we're doing with environment variables. We could just get rid of `$TARGET` because I think we don't really use it, and simply have one spot in `run_argument_parser()` to read `$CIRCLE_JOB` and fill in the info.